velox/exec/Operator.h

```diff
@@ -455,7 +455,7 @@ class Operator : public BaseRuntimeStatWriter {
 
   /// Returns a list of identity projections, e.g. columns that are projected
   /// as-is possibly after applying a filter.
-  const std::vector<IdentityProjection>& identityProjections() const {
+  virtual const std::vector<IdentityProjection>& identityProjections() const {
```

Comment:
Let's not make this change.

If we do, then this should be a separate PR and identityProjections_ needs to become private.

Reply:
Has removed.